mixer/pkg/server/server.go

```diff
@@ -254,3 +260,9 @@ func (s *Server) Close() error {
 func (s *Server) Addr() net.Addr {
 	return s.listener.Addr()
 }
+
+// GetDispatcherForTesting returns the dispatcherForTesting that was created during server creation. This should only
+// be used for testing purposes only.
+func (s *Server) GetDispatcherForTesting() mixerRuntime.Dispatcher {
```

Comment:
I think I'd rather drop the "ForTesting" suffix on the function and the field name. The comment is clear enough.